classes/local/form/verification_field.php

```diff
@@ -43,11 +43,16 @@ public function __construct($attributes=null, $submit = true) {
         }
 
         $attributes['autocomplete'] = 'one-time-code';
-        $attributes['autofocus'] = 'autofocus';
         $attributes['inputmode'] = 'numeric';
         $attributes['pattern'] = '[0-9]*';
         $attributes['class'] = 'tool-mfa-verification-code';
 
+        // If we aren't autosubmitting, this might be part of a larger form such as for setup.
```

Comment:
What did you mean by _autosubmitting_ . This comment may need to be cleaned up slightly if I'm reading that wrong. 

Perhaps:
```suggestion
        // If this is part of a larger form, such as for setup, then we shouldn't apply autofocus.
```